src/core/InterleavedBufferAttribute.js

```diff
@@ -17,7 +17,7 @@ THREE.InterleavedBufferAttribute.prototype = {
 
 	get length() {
```

Comment:
Related change for ```BufferAttribute``` https://github.com/mrdoob/three.js/commit/d5281a3a4bdac8a3b45be9e1fd562ab2b4ad9339